javascript/packages/fury/lib/classResolver.ts

```diff
@@ -36,7 +36,8 @@ export default class SerializerResolver {
   };
 
   private readStringPool: string[] = [];
-  private writeStringIndex: string[] = [];
+  private writeStringCount = 0;
+  private writeStringIndex: Record<string, number> = {};
```

Comment:
I have a idea, Perhaps, we can generate tag accessors code at runtime, 
```
```


I have an idea. Perhaps we can generate tag accessor code at runtime. Here's how we might do it:


1. Add a counter to classResolver:
```
writeStringCount = 0;
```

2. Generate a closure when registering a tag:

```

function createTagWriter() {
    retindex = -1;

    return function () {
        // store the index
        if (index > -1) {
            writeTagIndex(); // USESTRINGID

        } else {
            writeTagContent(); // USESTRINGVALUE
            index = writeStringCount++;
        }
    } 
}

```

3. When writing the tag, we can directly call the result of the createTagWriter function, which can help us avoid the hashmap lookup.